dask/dataframe/io/parquet.py

```diff
@@ -74,6 +74,11 @@ def _read_fastparquet(fs, paths, myopen, columns=None, filters=None,
     elif len(minmax) > 1:
         if index:
             index_col = index
+        elif index is None:
+            # attempt auto-index
+            index = pf._get_index()
+            if index:
+                index_col = index
         elif 'index' in minmax:
             index_col = 'index'
```

Comment:
For files that were not generated from pandas.DataFrame, it might be better to either explicitly indicate the index column or opt-in to inference based on column statistics. Having this inference on non-pandas data enabled by default seems a bit magical to me.